.github/workflows/android.yml

```diff
@@ -5,7 +5,7 @@ on:
     branches: [ "develop" ]
   pull_request:
     branches: [ "develop" ]
-
```

Comment:
No biggie, but if you have other edits, consider removing the whitespace (unless it's needed to work around a bug or something)

Reply:
The white space was intended. I was there to remove the feature branch name from the check and added the missing white space.